### DISCUSSION TITLE
Degree symbol for displaying temperatures

### DISCUSSION THREAD

**Discussion by ghost** [Bodmer/TFT_eSPI#1383] [Q&A - Fonts]
I have an ESP32 with an attached temp sensor and a tft display. I send the temp data to AdafruitIO for web display. Apparently the AdafruitIO default font recognize extended aschii, because it correctly displays char 248 as a degree symbol in a text field sent from the ESP32. The free fonts provided in the TFT_eSPI library stop at char 127, of course, so nothing is displayed on the TFT when the same text string is sent to it.
Has anyone developed a way to display that symbol on a TFT? Can anyone suggest font editing software to populate char 248 (or simply replace one of the 0 to 127 chars) in a given free font with a degree symbol?

**Comment by bobwolff68** [comment]
My solution to this very problem was to use a lower-case 'o' in a smaller font positioned up a half line kinda like a superscript:
```
// DEGREE
pCanvas->setFreeFont(FSSB9);
pCanvas->setTextDatum(TL_DATUM);
pCanvas->drawString("o", _w/2 + xpos/2 -4, 0);
```

**Reply by TangerineDreamer** [reply]
There is this natural font : "°"
You can find it with the key " ) ] ° " with the combination of left shift
The keyboard is a standard AZERTY
